src/components/Map/index.js

```diff
@@ -77,11 +114,43 @@ const MapView = ({coordinates = null, zoomLevel = DEFAULT_ZOOM}) => {
     if (!coordinates || coordinates.length !== 2) return;
     if (!coordinates.filter(value => isFinite(value)).length === 2) return;
     map.setView(coordinates, zoomLevel);
-  });
+    // eslint-disable-next-line
+  }, [coordinates]);
```

Comment:
I don't think that disabling the linter is the way to go here, the `zoomLevel` prop is used so if it changes, the map view should be updated accordingly.

Additionally, the `map` itself is also a dependency, if that changes (in the parent component for example), then this component *also* has to update or you may even get memory leaks.